incompatibility/incompatibility-report/incomp-scanning.go

```diff
@@ -43,24 +45,42 @@ func CreateIncompReport(env *plugins.Environment, reportType Report) {
 		}
 		openAPIdocument := &openapiv3.Document{}
 		err := proto.Unmarshal(model.Value, openAPIdocument)
-		if err != nil {
-			continue
-		}
-		printIncompatibilityReport(ScanIncompatibilities(openAPIdocument))
+		env.RespondAndExitIfError(err)
+		incompatibilityReport := ScanIncompatibilities(openAPIdocument)
+		writeProtobufMessage(incompatibilityReport, env)
+		env.RespondAndExit()
+	}
+	env.RespondAndExitIfError(errors.New("no supported models for incompatibility reporting"))
+}
 
+func writeProtobufMessage(incompatibilityReport *IncompatibilityReport, env *plugins.Environment) {
+	incompatibilityReportBytes, err := proto.Marshal(incompatibilityReport)
```

Comment:
Does proto.Marshal default to binary?  Do we want a readable format?

Reply:
yeah I was thinking that there could be further parameters to indicate if the user wants a human readable representation or not. But we need a binary representation so that the external tools could use the incompatibility report

Reply:
There are facilities to support ingesting text serialized protos.   I'm not sure if gnostic supports this across its infrastructure.

Reply:
now using prototext library